### PR TITLE
Fixes #25285: API documentation for pending nodes is missing an endpoint and fields

### PR DESCRIPTION
**Pull request by clarktsiory**
https://issues.rudder.io/issues/25285

image

### CODE REVIEW

**Comment by clarktsiory** [comment]
PR updated with a new commit

File: webapp/sources/api-doc/paths/nodes/pending-update.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# SPDX-License-Identifier: CC-BY-SA-2.0
+# SPDX-FileCopyrightText: 2013-2024 Normation SAS
+post:
+  summary: Update pending Nodes status
```
Comment (ElaadF) [inline webapp/sources/api-doc/paths/nodes/pending-update.yml:4]:
```suggestion
  summary: Update pending nodes status
```

**Comment by clarktsiory** [comment]
PR updated with a new commit

File: webapp/sources/api-doc/paths/nodes/pending-update.yml (hunk 1 above)
Comment (clarktsiory) [reply webapp/sources/api-doc/paths/nodes/pending-update.yml]: have a nice sleep @amousset :wink: : 
https://github.com/Normation/rudder/pull/5800/commits/d10633a5028dab54de1f0fc6c341cbef53043d51

**Review by ElaadF — APPROVED** [review]

**Comment by clarktsiory** [comment]
OK, squash merging this PR

**Comment by clarktsiory** [comment]
OK, merging this PR

**Pull request merged**